analyzers/tests/SonarAnalyzer.UnitTest/LiveVariableAnalysis/RoslynLiveVariableAnalysisTest.cs

```diff
@@ -766,6 +766,58 @@ public void Using_LiveInUntilTheEnd(string usingStatement, string suffix)
             context.Validate(context.Cfg.Blocks[6]);
         }
 
+        [TestMethod]
+        public void LocalFunctionInvocation_LiveIn()
+        {
```

Comment:
Minor - try to add the example when local function declared in the front of the code.